### ISSUE TITLE
H18 compatibility , Python support 

### ISSUE THREAD

**Issue by MirHadi-jam** [paulwinex/pw_Houdini_VEX_Editor#9]
Hello
This is a must have script editor IMO.
would you please make it H18 compatible and python support??

very generous of you.
thank you very much

**Comment by paulwinex** [comment]
Hi. I will try)

**Comment by MostHated** [comment]
Any word on this? This looks fantastic. Would love to use it!

**Comment by paulwinex** [comment]
Yes. I was try to fix bugs, but I came to the conclusion that i need to remake everything from scratch( Some widgets crash houdini because API is changed. I need to complete my current projects and then i can start new version of Vex editor
